main/src/main/scala/sbt/Defaults.scala

```diff
@@ -3344,7 +3345,12 @@ object Classpaths {
         p1.extra(SbtPomExtraProperties.VERSION_SCHEME_KEY -> x)
       case _ => p1
     }
-    p2
+    val p3 = releaseNotesURL.value match {
+      case Some(u) =>
+        p1.extra("info.releaseNotesUrl" -> u.toExternalForm)
```

Comment:
I assume this constant should be added to `SbtPomExtraProperties`, where is that exactly?

Reply:
https://github.com/sbt/librarymanagement/blob/develop/core/src/main/java/sbt/internal/librarymanagement/mavenint/SbtPomExtraProperties.java

Reply:
- https://github.com/sbt/librarymanagement/pull/410